app/users/views.py

```diff
@@ -15,7 +15,7 @@
 from django.utils.translation import ugettext_lazy as _
 from django.views.decorators.http import require_http_methods
 
-from projects.services import send_email
```

Comment:
The service was not working because I think there was a recursion between the 2 `send_email` functions

Reply:
Maybe I don't get your point, but in my understanding, renaming `send_email` to `send_email_exchange` makes no difference because there is only one `send_email` function. Django's function is named `send_mail` without an `e`. So I don't see why there should be a recursion happening. But I'm totally OK to leave it like this (maybe then it's even clearer). Thanks for the translation wrappers!

Reply:
I tried and the email was not sent, then I renamed it differently and the email was sent. It might also be due to confusion in testing as I was repeating the same steps many times in parallel with my local version and with the dev server online, I might have overlooked the url and though I was using the local version while I was using the not yet updated online dev version.